src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-containerized-function-monitoring/enable-containerized-function.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Create and deploy New Relic Lambda image layers
+metaDescription: After enabling New Relic's containerized Lambda function, you can create a layer as an image.
+redirects:
+freshnessValidatedDate: never
+---
+
+With containerized Lambda functions, you can create Lambda layers using container images. As an alternative to deploying layers as .zip archives, container images allow for increased flexibility and improved security.
```

Comment:
Not needed for merge I think, but the intro tells me nothing about NR's value prop here. Is this just a lambda thing we're documenting? Where does NR come in??

Reply:
A solid point, but not something I can address at the mo. It'll need to be a fast follow I think